stdlib.go

```diff
@@ -70,17 +77,25 @@ func NewStdlibAdapter(logger Logger, options ...StdlibAdapterOption) io.Writer {
 		timestampKey: "ts",
 		fileKey:      "caller",
 		messageKey:   "msg",
+		expectCaller: true,
 	}
 	for _, option := range options {
 		option(&a)
 	}
+
+	if a.expectCaller {
+		a.logRegexp = regexp.MustCompile(logRegexpDate + logRegexpTime + logRegexpFile + logRegexpMsg)
+	} else {
+		a.logRegexp = regexp.MustCompile(logRegexpDate + logRegexpTime + logRegexpMsg)
+	}
```

Comment:
Since these two regexp's are always built the same from constants should we should build them once at package scope like before?

Reply:
In combination with regexp option, that fits better here.